packages/nimble-components/src/banner/template.ts

```diff
@@ -0,0 +1,42 @@
+import { html, slotted, when } from '@microsoft/fast-element';
+import { DesignSystem } from '@microsoft/fast-foundation';
+import type { Banner } from '.';
+import { Button } from '../button';
+import { IconExclamationMark } from '../icons/exclamation-mark';
+import { IconInfo } from '../icons/info';
+import { IconTriangleFilled } from '../icons/triangle-filled';
+import { IconXmark } from '../icons/xmark';
+
+// prettier-ignore
+export const template = html<Banner>`
+    <template role="status" aria-label=${x => x.getAriaLabel()}>
+        <div class="icon">
+            ${when(x => x.severity === 'error', html`
+                <${DesignSystem.tagFor(IconExclamationMark)}></${DesignSystem.tagFor(IconExclamationMark)}>
+            `)}
+            ${when(x => x.severity === 'warning', html`
+                <${DesignSystem.tagFor(IconTriangleFilled)}></${DesignSystem.tagFor(IconTriangleFilled)}>
+            `)}
+            ${when(x => x.severity === 'info', html`
+                <${DesignSystem.tagFor(IconInfo)}></${DesignSystem.tagFor(IconInfo)}>
+            `)}
+        </div>
+        <div class="text">
```

Comment:
When I played with the storybook in Firefox on macOS with Voiceover turned on, I noticed it read "Title Text" several times. It looks like that text appears in the accessibility tree multiple times too. Can you check to see if it's behaving as expected?

It also read the title of the Close button multiple times but that doesn't appear multiple times in the accessibility tree so I suspect that was a Voiceover quirk.

Lastly, I'm not sure of the right way to verify "polite" status updates are working as expected but I did verify that voiceover made an announcement of the banner contents when I toggled the banner from closed to open.

![image](https://user-images.githubusercontent.com/10500124/217650907-82786d36-c50c-4bb7-a4b4-610179ab9f80.png)

Reply:
> It looks like that text appears in the accessibility tree multiple times too.

I think that is correct. The banner (with role status) has the name/label "Title text", which was intentional. Then there is also that static text "Title text". 

I tested with NVDA, and it doesn't seem to announce the banner itself. Not when I hover over it, and not when it initially becomes displayed. It seems like Voiceover does a better job. Except NVDA announces the close button as "clickable clickable close button" when I tab to it. It doesn't announce anything when I point to it with the mouse.
![image](https://user-images.githubusercontent.com/7282195/217672522-cc03e40d-2b29-4845-bd07-ea2ad462e73e.png)

Reply:
Looks like NVDA ignoring polite messages might be a known issue: https://github.com/nvaccess/nvda/issues/12464

Reply:
I've changed it so we're using `aria-labelledby` instead of `aria-label`. That should avoid the duplicate announcement, I think.